src/dune_pkg/opamUrl0.ml

```diff
@@ -29,19 +29,13 @@ let is_version_control t =
 ;;
 
 let is_local t = String.equal t.transport "file"
+let is_archive t = Option.is_some (Archive_driver.choose_for_filename t.path)
```

Comment:
```suggestion
let is_supported_archive t = Option.is_some (Archive_driver.choose_for_filename t.path)
``
We should be more specific that this is because it is an archive that we support.